hadoop-hdfs-project/hadoop-hdfs/src/main/java/io/hops/transaction/lock/LockFactory.java

```diff
@@ -297,7 +296,11 @@ public Lock getHashBucketLock(int storageId) {
   public Lock getLastBlockHashBucketsLock(){
     return new LastBlockReplicasHashBucketLock();
   }
-  
+
+  public Lock lockAllUsedHashBuckets() {
```

Comment:
can you use the same naming convention as for the remaining of the class: getHasBucketsLocksAllFileBlocks, it make it easier to use later on.